deploy/charts/burrito/templates/tenant.yaml

```diff
@@ -0,0 +1,62 @@
+{{- range $tenant := .Values.tenants }}
```

Comment:
I think we need a defaulting mechanism here as it would lead users to have to set default values on unused fields.
A merge between a default tenant map and each tenant should do the trick don't you think ?